app/models/bulk_submission.rb

```diff
@@ -400,7 +417,9 @@ def prepare_order(details) # rubocop:todo Metrics/CyclomaticComplexity
       attributes[:assets] = found_assets
     elsif found_assets.present? && found_assets != attributes[:asset_group].assets
       raise StandardError,
+            # rubocop:todo Layout/LineLength
```

Comment:
I'm going to stop commenting on these now.  There are more than I expected.  I know they're auto-generated -- shame the auto-generator isn't a bit smarter.